### PR TITLE
Update Texts and Info Pages

### LINKED ISSUE #127: Ricardian Contract Page
add ricardian contract component

### LINKED ISSUE #132: About Page

### PR DESCRIPTION
**Pull request by xavier506**
### What does this PR do?

- Resolve #127 
- Resolve #132 

### Steps to test

1. make run 
2. see About / Terms / Help pages

#### CheckList

- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check

### CODE REVIEW

**Review by adriexnet — APPROVED** [review]
LGTM

File: webapp/src/routes/Terms/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            {t('title')}
+          </Typography>
+        </CardContent>
+      </Card>
+    </Box>
+    // <RicardianContract
+    //   httpEndpoint={httpEndpoint}
+    //   contractName={contractName}
+    //   actionName={actionName}
+    //   showClauses={showClauses}
```
Comment (adriexnet) [inline webapp/src/routes/Terms/index.js:31]: We should avoid commit commented code

**Review by adriexnet — CHANGES_REQUESTED** [review]

File: webapp/src/routes/Terms/index.js (hunk 1 above)
Comment (tetogomez) [reply webapp/src/routes/Terms/index.js:31]: lol

**Review by tetogomez — COMMENTED** [review]
LGTM

File: webapp/src/routes/Terms/index.js (hunk 1 above)
Comment (xavier506) [reply webapp/src/routes/Terms/index.js:31]: 🆗

**Pull request closed without merging**